runtime/src/globals/mod.rs

```diff
@@ -12,7 +12,7 @@ pub mod console;
 pub mod encoding;
 #[cfg(feature = "fetch")]
 pub mod fetch;
-mod file;
+pub mod file;
```

Comment:
I think this is causing clippy warning, probably it only applies to externally visible structs.